report/src/sections/01-introduction.tex

```diff
@@ -40,10 +47,19 @@ \subsection{Non-terminating Analysis}\label{subsec:non-terminating-analysis}
     \label{lst:motivate-program}
 \end{listing}
 
+\begin{listing}
+    \begin{minted}{sql}
+    INSERT INTO account VALUES ("name", 1);
+    \end{minted}
+    \caption{A simple SQL statement inserting a tuple into the account table.}
+    \label{lst:motivate-sql-insert}
+\end{listing}
+
+
+Constantly changing database contexts under Cortesi and Halder's framework may lead to analysis termination issues. \todo{Gabriela, are you sure about this?, wouldn't the definition given on page 145 of (Halder Cortesi), (abstract insert) be used for this and since it is based on set union, then reach a fixed point?}
```

Comment:
the previous version was better:

`Because the database context would continually change under Cortesi and Halder, the analysis would not terminate.`